src/components/molecules/VenueRoomItem/VenueRoomItem.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useMemo } from "react";
+import { Button, Form, Modal } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { useAsyncFn } from "react-use";
+
+import { DEFAULT_VENUE_LOGO } from "settings";
+
+import { createRoom, createVenue_v2 } from "api/admin";
+
+import { venueInsideUrl } from "utils/url";
+import { buildEmptyVenue } from "utils/venue";
+
+import { RoomTemplate, VenueRoomTemplate } from "types/rooms";
+
+import { useUser } from "hooks/useUser";
+import { useVenueId } from "hooks/useVenueId";
+import { useShowHide } from "hooks/useShowHide";
+
+import {
+  venueRoomSchema,
+  roomSchema,
+} from "pages/Admin/Details/ValidationSchema";
+
+import { InputField } from "components/atoms/InputField";
+
+import "./VenueRoomItem.scss";
+
+export interface VenueRoomItemProps {
+  icon: string;
+  text: string;
+  template?: VenueRoomTemplate;
+}
+
+export const VenueRoomItem: React.FC<VenueRoomItemProps> = ({
+  icon,
+  text,
+  template,
+}) => {
+  const {
+    isShown: isModalVisible,
+    show: showModal,
+    hide: hideModal,
+  } = useShowHide();
+
+  const { user } = useUser();
+
+  const venueId = useVenueId();
+
+  const isVenuePortal = template !== RoomTemplate.external;
+
+  const { register, getValues, handleSubmit, errors } = useForm({
+    validationSchema: isVenuePortal ? venueRoomSchema : roomSchema,
+    defaultValues: {
+      roomTitle: "",
+      roomUrl: "",
+      venueName: "",
+      template: template,
```

Comment:
do you need to specify empty default values?

Reply:
If I don't all of these values will fallback to `undefined` and I need to check for the fields if they exists or not, setting them empty string is easier and the end result is the same.